docs/end-user/policies/apply-once.md

```diff
@@ -29,6 +30,40 @@ spec:
         enable: true
 ```
 
-In this case, if you change the replicas of the hello-world deployment after Application enters running state, the change will not be reverted. On the contrary, if you disable the apply-once policy (by default), any changes to the replicas of hello-world application will be brought back in the next reconcile loop.
+In this case, if you change the replicas or other configurations of the hello-world deployment after Application enters the running state, the change will not be reverted. On the contrary, if you disable the apply-once policy (by default), any changes to the replicas of the hello-world application will be brought back in the next reconcile loop.
```

Comment:
`or other configurations` is not correct. We can only control the fields that do not allow drift. Field `replicas` is one of them, but the rest are not necessarily are.

Reply:
change to "if you change the replicas or other managed fields of the hello-world deployment"